framework/src/util/CFRLogger.cpp

```diff
@@ -21,25 +21,22 @@
  =============================================================================*/
 
 #include "CFRLogger.h"
+#include "CoreBundleContext.h"
+#include "cppmicroservices/GetBundleContext.h"
 
 namespace cppmicroservices {
 namespace cfrimpl {
-CFRLogger::CFRLogger(cppmicroservices::BundleContext context)
-  : cfrContext(std::move(context))
-  , serviceTracker(
-      std::make_unique<cppmicroservices::ServiceTracker<
-        cppmicroservices::logservice::LogService>>(cfrContext, this))
+CFRLogger::CFRLogger(cppmicroservices::CoreBundleContext*)
```

Comment:
`cppmicroservices::CoreBundleContext*` is never used. this can be removed as a parameter.